src/elements/hx-file-icon/index.spec.js

```diff
@@ -0,0 +1,154 @@
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered Light DOM should NOT equal simple template ${template}`, async () => {
+            const component = /** @type {HXFileIconElement} */ await fixture(template);
+
+            expect(component).lightDom.to.not.equal(template);
+        });
+
+    });
+
+    describe('test Shadow DOM', () => {
+        describe('verify render', () => {
+            it('should have a static Shadow DOM', async function () {
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+
+                // FIXME: temp fix issue submitted https://github.com/open-wc/open-wc/issues/1540
+                const shadow = component.shadowRoot.innerHTML;
+
+                expect(component).shadowDom.to.equal(shadow);
+            });
+
+            it('should render the Shadow Root mode open', async () => {
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+                const mode = component.shadowRoot.mode;
+
+                expect(mode).to.equal("open");
+            });
+
+            it('should have a single <slot>', async () => {
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelectorAll('slot');
+                const len = query.length;
+
+                expect(len).to.be.equal(0);
+            });
+
+            it('should have an unnamed <slot>', async () => {
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+                const name = component.slot;
+
+                if ( name !== null ) {
+                    expect(name).to.be.equal('');
+                } else {
+                    expect(name).to.be.null;  // IE11, Legacy Edge, and older browsers
+                }
+            });
+        });
+
+        describe('verify Shadow DOM markup', () => {
+            it('markup should contain a #hxFileIcon <div>', async () => {
+                const elSelector = 'div#hxFileIcon';
+                const id = 'hxFileIcon';
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain a #hxBase with an file type <hx-icon>', async () => {
+                const elSelector = 'div#hxFileIcon > hx-icon';
+                const Type = 'file';
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.type;
```

Comment:
Please change to variable to `queryType`.

Reply:
Changes have been done, Please review